src/transformers/models/fsmt/modeling_fsmt.py

```diff
@@ -658,9 +659,16 @@ def __init__(self, config: FSMTConfig, embed_tokens: nn.Embedding):
             [DecoderLayer(config) for _ in range(config.decoder_layers)]
         )  # type: List[DecoderLayer]
 
+        if is_deepspeed_zero3_enabled():
+            import deepspeed
+
+            with deepspeed.zero.GatheredParameters(self.embed_tokens.weight, modifier_rank=None):
+                embed_tokens_weight_shape = self.embed_tokens.weight.shape
+        else:
+            embed_tokens_weight_shape = self.embed_tokens.weight.shape
         self.output_projection = nn.Linear(
-            self.embed_tokens.weight.shape[1],
-            self.embed_tokens.weight.shape[0],
+            embed_tokens_weight_shape[1],
+            embed_tokens_weight_shape[0],
             bias=False,
```

Comment:
Looks like it should fit in one line now?

Reply:
It actually does fit if I change it manually. Thank you!

I had no idea that auto-formatter doesn't do the work. Shouldn't it unwrap lines that don't need wrapping automatically?

Reply:
No the auto-formatter does not re-put stuff back on one line, sadly.